.github/workflows/async-update.yml

```diff
@@ -0,0 +1,82 @@
+---
+name: Update when async branch is pushed
+
+'on':
+  workflow_call:
+
+permissions:
+  pull-requests: write   # to create labels, see https://docs.github.com/en/rest/issues/labels?apiVersion=2022-11-28#create-a-label
+
+jobs:
+  update:
+    name: Update everything
+    runs-on: ubuntu-latest
+    # only run if the created branch or tag starts with the correct pattern, namely "async-"
+    if: startsWith(github.ref, 'refs/heads/async-v')
+    steps:
+      - name: Update, remove labels, publish table
+        env:
+          GH_TOKEN: ${{ github.token }}
+          ref: ${{ github.ref }}
+
+        id: create_label
+        run: |
+          branch_name="${ref##*/}"
+
+          # we check now if this branch contains the tag <branch-name>-start which is the one where we branched off the default back back in the days
+          start_tag="${branch_name}-start"
+          if ! GIT_DIR=${GITHUB_REPOSITORY} git branch --contains "${start_tag}" | grep "${branch_name}" ; then
```

Comment:
Shouldn't you make sure that branch_name is exactly matched?

Reply:
Yes. done now